gateway/constants.ts

```diff
@@ -1,2 +1,11 @@
-export const PORT = 7000;
-export const HOST = "0.0.0.0";
+import dotenv from "dotenv";
+dotenv.config();
+
+const port = process.env.PORT || 7000;
+const host = process.env.HOST || "0.0.0.0";
```

Comment:
Was this the const that was causing problems for us? So now it is set to use the env OR 0.0.0.0 correct?

Reply:
kinda, host still needs to be 0.0.0.0 for when the app starts to listen but there is a string just under the listen that gets calculated depending on the host. but it will happen at the end 😅

Reply:
as long as it fixes it, I don't care what kind of voodoo it does.